CONTRIBUTING.md

```diff
@@ -0,0 +1,12 @@
+# Contributing to [Project Name]
```

Comment:
It's a good idea to add a Contributing guide, but this one is too generic. Also, the relevant content is already in the readme.